Fixed DATETIME if object

chemitaxis opened the pull request:
Please review #15

cognitom commented on lib/types.js:
```diff
@@ -70,7 +78,7 @@ const NATIVE_TYPES = {
  * @returns {number|null} Timestamp in msec or null if not a valid value
  */
-function getValueOfDate (str) {
+function getValueOfDate(str) {
```
pls keep the space.

cognitom commented on lib/types.js:
```diff
@@ -84,7 +92,7 @@ function getValueOfDate (str) {
  * @returns {string|null} time or null
  */
-function normalizeTime (str) {
+function normalizeTime(str) {
```
pls keep the space here, too.

cognitom commented on lib/types.js:
```diff
@@ -94,4 +102,4 @@ function normalizeTime (str) {
 }
 
-module.exports = NATIVE_TYPES
+module.exports = NATIVE_TYPES
```
pls keep the trailing new-line.

chemitaxis commented:
Hi, I have fixed. I have added .eslintrc file for other contributors. I have added dependecies for all necessary. Works with eslint ;) https://github.com/standard/eslint-config-standard

cognitom commented:
@chemitaxis could you make separate PR about `eslint`? It seems better to focus the issue, I think. But, honestly, I don't want to add extra dependencies...

Anyway, thanks for your contribution!

chemitaxis commented:
Hi, no problem with eslint devDependencies, I have removed. Here you have all the code in the PR (https://github.com/cognitom/momy/pull/16/files)

cognitom commented:
@chemitaxis thanks a lot! I'll publish a new version to npm soon 😄

cognitom commented:
Ooops, this PR can't pass our tests. @chemitaxis do you have any fix for them?

```
Momy Types
    ✓ BIGINT
    2) DATE
    3) DATETIME
    ✓ DOUBLE
    ✓ TIME
    ✓ TINYINT
    ✓ VARCHAR
    ✓ TEXT


  20 passing (37s)
  3 failing

  1) Momy: fieldCase camel:
     TypeError: Cannot read property 'field1' of undefined
      at Context.<anonymous> (test/specs/field-case.js:37:23)
      at Generator.next (<anonymous>)
      at onFulfilled (node_modules/co/index.js:65:19)
      at <anonymous>
      at process._tickCallback (internal/process/next_tick.js:169:7)

  2) Momy Types DATE:

      AssertionError [ERR_ASSERTION]: '"Invalid date"' == 'NULL'
      + expected - actual

      -"Invalid date"
      +NULL
      
      at Context.it (test/specs/types.js:20:12)

  3) Momy Types DATETIME:

      AssertionError [ERR_ASSERTION]: '"Invalid date"' == 'NULL'
      + expected - actual

      -"Invalid date"
      +NULL
      
      at Context.it (test/specs/types.js:29:12)
```

chemitaxis commented:
Ok, I'm working now on this...

cognitom commented:
@chemitaxis thank you for your quick fix 😉

chemitaxis commented:
You are welcome!
